New Synthetics home page (#7771)
DataDog/documentation: content/en/synthetics/_index.md

  tag: "Documentation"
  text: "Synthetics Guides"
- link: "/synthetics/troubleshooting/"
  tag: "Documentation"
  text: "Troubleshooting Synthetics"
---

{{< img src="synthetics/synthetics_home.png" alt="Synthetics home page" >}}

Synthetic tests allow you to observe how your systems and applications are performing using **simulated requests and actions from around the globe**. Datadog **tracks the performance of your webpages and APIs** from the backend to the frontend, and at various network levels (HTTP, TCP, SSL) in a controlled and stable way, alerting you in case of faulty behavior (regression, broken feature, high response time, unexpected status code, etc.). **End-to-end testing production and CI environments** increases development teams’ speed as it puts an end to the fear of defective code making it to production. **Computing SLOs** on your key endpoints and user journeys makes it easier to stick to your application performance targets and ultimately provide a consistent customer experience. 

## Getting Started
Create your first Synthetic test and start monitoring web applications to improve their performance in just a few minutes.
### Set up API tests

> API tests allow you to launch requests and perform verifications on your web applications at various network levels: HTTP, TCP, SSL. Create your first [HTTP test][1], [TCP test][link to regular documentation], and [SSL test][link to regular documentation] to get started with API and network monitoring.

Reviewer: No available documentation for TCP tests for now (I need to create it) but we can link to https://docs.datadoghq.com/synthetics/api_tests/?tab=ssltest for SSL tests

API tests allow you to launch requests and perform verifications on your web applications at various network levels: HTTP, TCP, SSL. Create your first [HTTP test][1], TCP test, and [SSL test][6] to get started with API and network monitoring.

{{< img src="synthetics/api_test.png" alt="API tests"  style="width:100%;">}}

### Record Browser tests

Record end-to-end tests to monitor how your customers experience your webpages from around the world using [Synthetics Browser tests][2].

{{< img src="synthetics/browser_test.gif" alt="Browser tests"  style="width:100%;">}}

### Launch Private Locations

Use [Synthetics Private Locations][3] to monitor internal APIs and websites or to create custom locations in areas that are mission-critical to your business.

{{< img src="synthetics/private_locations.png" alt="Private locations"  style="width:100%;">}}

### Run tests with your integration and deployment processes

Leverage your Synthetic tests as [canaries][4] or run them directly within your [CI pipelines][4] to start shipping without fearing faulty code might impact your customers experience.

 {{< img src="synthetics/ci.png" alt="CI tests"  style="width:100%;">}}

> ### Connect Synthetics and Traces

Reviewer 2: ```suggestion
### Connect Synthetics and traces
```

title: Synthetics
kind: documentation

> description: "Make sure the most critical parts of your product are up and running from various locations around the world."

Reviewer: ```suggestion
description: "Use automated testing to ensure the most critical parts of your systems and applications are up and running from various locations around the world."
```

disable_toc: true
aliases:
  - /integrations/synthetics/
further_reading:
- link: "https://www.datadoghq.com/blog/introducing-synthetic-monitoring/"
  tag: "Blog"
  text: "User experience monitoring with Datadog Browser Tests"
- link: "https://www.datadoghq.com/blog/browser-tests/"
  tag: "Blog"
  text: "Best practices for creating end-to-end tests"
- link: "https://www.datadoghq.com/blog/test-creation-best-practices/"
  tag: "Blog"
  text: "Introducing Datadog Synthetics"
- link: "/synthetics/troubleshooting/"
  tag: "Documentation"
  text: "Troubleshooting Synthetics"
- link: "/synthetics/settings/"
  tag: "Documentation"
  text: "Synthetics Settings"
- link: "/synthetics/guide/"
  tag: "Documentation"
  text: "Synthetics Guides"
---

> {{< img src="synthetics/home_page.png" alt="Synthetics home page" >}}

Reviewer: Sorry, last change after review with GJ: https://a.cl.ly/7Kum8Ywr

> Synthetic tests allow you to observe how all your systems are performing using simulated user requests from around the globe. Datadog tracks the performance of your webpages and APIs at all network levels (HTTP, TCP, SSL) in a controlled and stable way, alerting you in case of faulty behavior (high response time, unexpected status code, etc.). Computing SLOs on your key endpoints and user journeys makes it easier to stick to your application performance targets and ultimately provide a consistent customer experience. By end-to-end testing both your production and CI environments, you can increase your development teams’ speed and stop fearing defective code might make it to production.

Reviewer: ```suggestion
Synthetic tests allow you to observe how all your systems are performing using **simulated user requests from around the globe**. Datadog **tracks the performance of your webpages and APIs** at all network levels (HTTP, TCP, SSL) in a controlled and stable way, alerting you in case of faulty behavior (high response time, unexpected status code, etc.). **End-to-end testing production and CI environments** increases development teams’ speed as it puts an end to the fear of defective code making it to production. **Computing SLOs** on your key endpoints and user journeys makes it easier to stick to your application performance targets and ultimately provide a consistent customer experience. 
```

## Getting Started
Create your first Synthetic test and start monitoring web applications to improve their performance in just a few minutes.
### Set up API tests

> API tests allow you to launch requests and perform verifications on your web applications at various network levels: HTTP, TCP, DNS. Create your first [HTTP test][1] and get started with API monitoring.

Reviewer: ```suggestion
API tests allow you to launch requests and perform verifications on your web applications at various network levels: HTTP, TCP, SSL. Create your first [HTTP test][1] and get started with API monitoring.
```

{{< img src="synthetics/api_test.png" alt="API tests"  style="width:100%;">}}

### Record Browser tests

Record end-to-end tests to monitor how your customers experience your webpages from around the world using [Synthetics Browser tests][2].

{{< img src="synthetics/browser_test.gif" alt="Browser tests"  style="width:100%;">}}

### Launch Private Locations

Use [Synthetics Private Locations][3] to monitor internal APIs and websites or to create custom locations in areas that are mission-critical to your business.

{{< img src="synthetics/private_locations.png" alt="Private locations"  style="width:100%;">}}

> ### Run tests from your CI

Reviewer: ```suggestion
### Run tests with your integration and deployment processes
```

>  Include your [Synthetic tests in your CI pipelines][4] and start shipping without fearing faulty code might make it to production.

Reviewer: ```suggestion
Leverage your Synthetic tests as [canaries][4] or run them directly within your [CI pipelines][4] to start shipping without fearing faulty code might impact your customers experience.
```

 {{< img src="synthetics/ci.png" alt="CI tests"  style="width:100%;">}}

### Connect Synthetics and Traces

Use the [out of the box integration between Synthetic tests and APM traces][5] to find the root cause of failures across frontend, network and backend requests.

{{< img src="synthetics/synthetics_traces.gif" alt="Synthetics" style="width:100%;">}}

> ## Further Reading

Reviewer: After GJ review, could we do:
1. Introducing Datadog Synthetics
2. User experience monitoring with Datadog Browser Tests
3. Best practices for creating end-to-end tests
4. Guides
5. Troubleshooting Synthetics

The Settings page is quite empty at the moment so we'd rather not link to it.

> - link: "https://www.datadoghq.com/blog/introducing-synthetic-monitoring/"

Reviewer: For the further reading section, I'm suggesting, in order:
https://www.datadoghq.com/blog/introducing-synthetic-monitoring/
https://www.datadoghq.com/blog/browser-tests/
https://www.datadoghq.com/blog/test-creation-best-practices/
https://docs.datadoghq.com/synthetics/guide/ (not many of them, but we're planning to add a lot more)

  tag: "Blog"
  text: "Introducing Datadog Synthetics"
- link: "/synthetics/troubleshooting/"
  tag: "Documentation"
  text: "Troubleshooting Synthetics"
- link: "/synthetics/settings/"
  tag: "Documentation"
  text: "Synthetics Settings"
---

> {{< img src="synthetics/synthetics_home_page.png" alt="Synthetics home page" >}}

Reviewer: Let's use https://a.cl.ly/JruLgjxb instead 
Also, do you think we could remove the table of content on the right like it was done for RUM? So that we can preview a larger image?

> Synthetic tests allow you to observe how all your systems are performing as experienced by your users from around the globe -  Datadog tracks the performance of your applications and API endpoints using simulated user requests and browser rendering. Synthetics helps you ensure uptime, identify regional issues, compute error budgets, automate tests, and track application performance. By unifying Synthetics with your metrics, traces, and logs, you can test your production environment end-to-end.

Reviewer: ```suggestion
Synthetic tests allow you to observe how all your systems are performing using simulated user requests from around the globe. Datadog tracks the performance of your webpages and APIs at all network levels (HTTP, TCP, SSL) in a controlled and stable way, alerting you in case of faulty behavior (high response time, unexpected status code, etc.). Computing SLOs on your key endpoints and user journeys makes it easier to stick to your application performance targets and ultimately provide a consistent customer experience. By end-to-end testing both your production and CI environments, you can increase your development teams’ speed and stop fearing defective code might make it to production.
```

## Getting Started

> 

Reviewer: Can we add a quick sentence like "Create your first Synthetic test and start monitoring web applications to improve their performance in just a few minutes"?

### Set up API tests

> Monitor how your customers from around the world experience your http API endpoints using [Synthetics API tests][1].

Reviewer: ```suggestion
API tests allow you to launch requests and perform verifications on your web applications at various network levels: HTTP, TCP, DNS. Create your first [HTTP test][1] and get started with API monitoring.
```

> {{< img src="synthetics/api.png" alt="API tests"  style="width:100%;">}}

Reviewer: That appears too small :'( let's use a screenshot of the context panel only: https://a.cl.ly/geuwqxG9

> ### Set up Browser tests

Reviewer: ```suggestion
### Record Browser tests
```

> Monitor how your customers from around the world experience your webpage using [Synthetics Browser tests][2].

Reviewer: ```suggestion
Record end-to-end tests to monitor how your customers experience your webpages from around the world using [Synthetics Browser tests][2].
```

> {{< img src="synthetics/browser.gif" alt="Browser tests"  style="width:100%;">}}

Reviewer: Let's use that shorter one instead: https://a.cl.ly/wbuWP1jy

> ### Set up Private Locations

Reviewer: ```suggestion
### Launch Private Locations
```

> Use [Synthetics Private Locations][3] to monitor internal APIs and websites - private URLs that aren’t accessible from the public internet.

Reviewer: ```suggestion
Use [Synthetics Private Locations][3] to monitor internal APIs and websites or to create custom locations in areas that are mission-critical to your business.
```

* Ranges are not supported
* Boosting is not supported
### Set up Browser tests

Finally, the following characters are reserved: `-`, `(`, `)`, `"`, `~`, `*`, `:`, `.`, and whitespace. To search check fields that include any of them, wrap the field string in quotes: `status:("OK") AND "doc-check"` is a valid query string; `status:("OK") AND doc check` is not.
Monitor how your customers from around the world experience your webpage using [Synthetics Browser tests][2].

## Create a check
{{< img src="synthetics/browser.gif" alt="Browser tests"  style="width:100%;">}}

Select **Create a New check +** in the upper right corner of the Synthetics page to create an [API tests][2] or a [browser test][3]. When a test fails, the endpoint is not retested: the uptime directly considers the endpoint as `down`.
### Set up Private Locations

{{< img src="synthetics/create_a_check.png" alt="Create a check"  style="width:80%;">}}
Use [Synthetics Private Locations][3] to monitor internal APIs and websites - private URLs that aren’t accessible from the public internet.

## Metrics
{{< img src="synthetics/private_locations.png" alt="Private locations"  style="width:100%;">}}

> The following metrics are generated by Synthetics checks:

Reviewer: Can we add a CI dedicated section, after the private location one and before the APM one?

Title: Run tests from your CI
Description: Include your Synthetic tests within your CI pipelines and start shipping without fearing faulty code might make it to production. (link to https://docs.datadoghq.com/synthetics/ci/)
Screenshot: https://a.cl.ly/mXuA6Rnx

Would it be possible to start indexing https://docs.datadoghq.com/synthetics/ci/ and add a link to reach out to support to the banner of that CI page as well?

> [Link simulated API tests][4] to traces to find the root cause of failures across frontend, network and backend requests.

Reviewer: ```suggestion
Use the [out of the box integration between Synthetic tests and APM traces][4] to find the root cause of failures across frontend, network and backend requests.
```

> {{< img src="synthetics/traces.gif" alt="Synthetics"  style="width:100%;">}}

Reviewer: Can we try with this zoom in version https://a.cl.ly/p9uPwdPO?
